### PR TITLE
Better documentation of draw_svg y_ticks

### PR DESCRIPTION
**Pull request by hyanwong**
See #2990

### CODE REVIEW

File: python/tskit/trees.py [hunk 1]
```diff
@@ -1885,9 +1885,11 @@ def draw_svg(
...
         :param str y_label: Place a label to the left of the plot. If ``None`` (default)
             and there is a Y axis,  create and place an appropriate label.
-        :param list y_ticks: A list of Y values at which to plot tickmarks (``[]``
-            gives no tickmarks). If ``None``, plot one tickmark for each unique
-            node value.
+        :param Union[list, dict] y_ticks: A list of Y values at which to plot
+            tickmarks, or a dictionary mapping Y values to labels (``[]`` gives no
+            tickmarks). If ``None`` (default), plot one tickmark for each unique node
+            value. Note that if ``time_scale="rank"``, the Y values refer to the
+            zero-based rank of the plotted nodes, rather that then node time itself.
```
Comment (jeromekelleher) [inline python/tskit/trees.py:1892]: s/then/the

File: python/tskit/trees.py [hunk 2]
```diff
@@ -7205,9 +7207,11 @@ def draw_svg(
...
         :param str y_label: Place a label to the left of the plot. If ``None`` (default)
             and there is a Y axis, create and place an appropriate label.
-        :param list y_ticks: A list of Y values at which to plot tickmarks (``[]``
-            gives no tickmarks). If ``None``, plot one tickmark for each unique
-            node value.
+        :param Union[list, dict] y_ticks: A list of Y values at which to plot
+            tickmarks, or a dictionary mapping Y values to labels (``[]`` gives no
+            tickmarks). If ``None`` (default), plot one tickmark for each unique node
+            value. Note that if ``time_scale="rank"``, the Y values refer to the
+            zero-based rank of the plotted nodes, rather that then node time itself.
```
Comment (jeromekelleher) [inline python/tskit/trees.py:7214]: ditto

**Review by jeromekelleher — APPROVED** [review]
Spotted typo

File: python/tskit/trees.py (hunk 1 above)
Comment (hyanwong) [reply python/tskit/trees.py:1892]: Good spot, thanks.

**Comment by hyanwong** [comment]
I also added the warning if ticks are outside the y axis.

**Comment by benjeffery** [comment]
@mergifyio rebase

**Comment by benjeffery** [comment]
@mergifyio rebase

**Pull request merged**